.gitpod.Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM gitpod/workspace-full:2022-11-09-13-54-49
 
 RUN mkdir -p ~/.local/bin
-RUN curl -L -o ~/.local/bin/soroban https://github.com/stellar/soroban-cli/releases/download/v0.2.1/soroban-cli-0.2.1-x86_64-unknown-linux-gnu
```

Comment:
Should we use https://github.com/stellar/soroban-tools vs https://github.com/stellar/soroban-cli?

Reply:
`https://github.com/stellar/soroban-tools/releases/download/v0.3.1/soroban-cli-0.3.1-x86_64-unknown-linux-gnu`

Reply:
We should, although it will still work, GitHub just redirects.